cdhweb/pages/tests/test_pages.py

```diff
@@ -71,3 +71,9 @@ def test_parent_pages(self):
     def test_subpages(self):
         # only allowed child is content page
         self.assertAllowedSubpageTypes(ContentPage, [ContentPage])
+
+
+class TestRelatedLinkType(WagtailPageTests):
+    def test_str(self):
+        restype = RelatedLinkType(name='twitter')
+        assert str(restype) == restype.name
```

Comment:
I think this can just be a `SimpleTestCase` rather than `WagtailPageTests` since it's not using any of that functionality. might want to also change the variable name "restype" since i think that's a holdover when when it was called "resource type", but no biggie

Reply:
For pytest, it actually doesn't need to extend anything if we're not using any functionality from the test case classes

Reply:
addressed by 6c2e39d0e679047e2728f65cb627db96b65e0405